### PR TITLE
feature/redesign

### PR DESCRIPTION
**Pull request by kemrankrm**

### CODE REVIEW

File: src/components/Intro/styles.module.scss [hunk 1]
```diff
@@ -1,35 +1,114 @@
...
+    opacity: 1;
+  }
 }
 
 .wrapper {
   display: flex;
-  max-width: 1280px;
+
+  max-width: 1200px;
+  height: 580px;
```
Comment (d-otynshin) [inline src/components/Intro/styles.module.scss:27]: add min-height

File: src/components/Dropdown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+export const Dropdown: FC<IDropdown> = ({ section, menuItems }) => {
+  return (
+    <ul className={styles['dropdown__list']}>
+      {menuItems.map((item: any) => {
+        return (
+          <li
+            className={cn(
+              styles['dropdown__item'],
+              styles[`dropdown__item--${section}`],
```
Comment (d-otynshin) [inline src/components/Dropdown/index.tsx:19]: remove

File: src/components/Features/index.tsx [hunk 3]
```diff
@@ -1,24 +1,87 @@
...
-              alt={item.imageAlt}
-              title={item.title}
-              text={item.description}
-            />
-          );
-        })}
+    <section className={styles['features']}>
+      <div className={styles['wrapper']}>
+        <div
+          className={cn(styles['features__about'], styles['features__card'])}
```
Comment (d-otynshin) [inline src/components/Features/index.tsx:13]: features__card--about

File: src/components/Header/styles.module.scss [hunk 4]
```diff
@@ -11,25 +11,31 @@
...
 
   border-bottom: 1px solid rgb(255, 255, 255, .2);
   box-sizing: border-box;
-  width: calc(100% - 80px);
+  max-width: 1200px;
+  min-height: 96px;
 
   margin: 0 auto;
 
   font-family: Fester, Inter, sans-serif;
```
Comment (d-otynshin) [inline src/components/Header/styles.module.scss:32]: remove old font

File: src/components/Intro/index.tsx [hunk 5]
```diff
@@ -1,20 +1,41 @@
...
-            Онлайн-сервис покупки авто <br /> из-за границы
-          </h1>
-          <h3 className={styles.subtitle}>
-            Безопасный, быстрый и удобный <br />
-            способ покупки и доставки авто в Казахстан
-          </h3>
+        <div className={styles['content']}>
+          <div className={styles['content__text']}>
+            <p className={styles['content__note']}>Первый и единственный</p>
+            <h1 className={styles['content__title']}>
```
Comment (d-otynshin) [inline src/components/Intro/index.tsx:10]: content__text-title, content__text-note

File: src/components/Dropdown/index.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+interface IDropdown {
+  section: string;
+  menuItems: any;
+}
+
+export const Dropdown: FC<IDropdown> = ({ section, menuItems }) => {
+  return (
+    <ul className={styles['dropdown__list']}>
+      {menuItems.map((item: any) => {
```
[the remark this answers is not in the document]
Comment (kemrankrm) [reply src/components/Dropdown/index.tsx:14]: done

File: src/components/Dropdown/index.tsx [hunk 7]
```diff
@@ -0,0 +1,31 @@
+import styles from './styles.module.scss';
+import Link from 'next/link';
+import { FC } from 'react';
+import cn from 'classnames';
+
+interface IDropdown {
+  section: string;
+  menuItems: any;
```
[the remark this answers is not in the document]
Comment (kemrankrm) [reply src/components/Dropdown/index.tsx:8]: done

File: src/components/NavItem/index.tsx [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleDropdownOpen = () => {
+    setIsOpen((isOpen) => !isOpen);
+  };
+
+  if (menuItems) {
+    return (
+      <li
+        className={styles['navitem']}
```
[the remark this answers is not in the document]
Comment (kemrankrm) [reply src/components/NavItem/index.tsx:29]: done

File: src/components/NavItem/index.tsx [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+  text: string;
+  menuItems?: Array<IMenuItem>;
+  section: string;
+  link: string;
+}
+
+export const NavItem: FC<INavItem> = ({ text, menuItems, section, link }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleDropdownOpen = () => {
```
[the remark this answers is not in the document]
Comment (kemrankrm) [reply src/components/NavItem/index.tsx:22]: done

File: src/components/JorneyMap/constants.tsx [hunk 10]
```diff
@@ -0,0 +1,27 @@
+import partnersPic from '@public/images/people.png';
+import navigationPic from '@public/images/navigation.png';
+import carPic from '@public/images/car-white-svgrepo-com.svg';
+
+export const advantageContent = [
+  {
+    number: 1,
```
[the remark this answers is not in the document]
Comment (kemrankrm) [reply src/components/JorneyMap/constants.tsx:7]: done

File: src/components/Intro/styles.module.scss (hunk 1 above)
Comment (kemrankrm) [reply src/components/Intro/styles.module.scss:27]: done

File: src/components/Dropdown/index.tsx (hunk 2 above)
Comment (kemrankrm) [reply src/components/Dropdown/index.tsx:19]: done

File: src/components/Features/index.tsx (hunk 3 above)
Comment (kemrankrm) [reply src/components/Features/index.tsx:13]: done

File: src/components/Header/styles.module.scss (hunk 4 above)
Comment (kemrankrm) [reply src/components/Header/styles.module.scss:32]: done

File: src/components/Intro/index.tsx (hunk 5 above)
Comment (kemrankrm) [reply src/components/Intro/index.tsx:10]: done

**Review by d-otynshin — APPROVED** [review]

**Pull request merged**